src/main/java/com/instana/operator/string/Contents.java

```diff
@@ -1,6 +1,6 @@
-package com.instana.operator.util;
+package com.instana.operator.string;
 
-public class StringUtils {
+public class Contents {
```

Comment:
🤔

Reply:
`StringContents`? I know you're trying to get away from Utils but just plain `Contents` doesn't seem to have enough gravitas... ;)

Reply:
All of the methods accept CharSequence's so named it CharSequenceContent. Not in love with it but 🤷‍♂